### PR TITLE
LibGUI+Userland: Add an action to access the command palette

### PR DESCRIPTION
**Pull request by demostanis**
This PR adds `make_search_action` to add an action to quickly access the command palette, since its keybinding isn't easily memorable.
image

### CODE REVIEW

File: Userland/Libraries/LibGUI/CommandPalette.cpp [hunk 1]
```diff
@@ -253,7 +253,7 @@ void CommandPalette::collect_actions(GUI::Window& parent_window)
                 collect_actions_from_menu(*menu_item.submenu());
 
             auto const* action = menu_item.action();
-            if (action && action->is_enabled())
+            if (action && action->is_enabled() && action->text() != "Search...")
```
Comment (alimpfard) [inline Userland/Libraries/LibGUI/CommandPalette.cpp:256]: This seems overly generic, doing this for _any_ "Search..." action can be unexpected

Comment (demostanis) [reply Userland/Libraries/LibGUI/CommandPalette.cpp:256]: Should I check for the menu name too or is there a better way to identify an action?

Comment (demostanis) [reply Userland/Libraries/LibGUI/CommandPalette.cpp]: I made it check the shortcut instead.

**Comment by demostanis** [comment]
that looks like CI flake

File: Userland/Libraries/LibGUI/CommonActions.cpp [hunk 2]
```diff
@@ -197,6 +198,22 @@ NonnullRefPtr<Action> make_rotate_counterclockwise_action(Function<void(Action&)
     return GUI::Action::create("Rotate &Counterclockwise", { Mod_Ctrl | Mod_Shift, Key_LessThan }, Gfx::Bitmap::try_load_from_file("/res/icons/16x16/edit-rotate-ccw.png"sv).release_value_but_fixme_should_propagate_errors(), move(callback), parent);
 }
 
+NonnullRefPtr<Action> make_search_action(Window* window)
+{
+    auto action = Action::create("&Search...", { Mod_Ctrl | Mod_Shift, Key_A }, MUST(Gfx::Bitmap::try_load_from_file("/res/icons/16x16/find.png"sv)), [=](auto&) {
```
Comment (linusg) [inline Userland/Libraries/LibGUI/CommonActions.cpp:203]: "Search" is not the right term for this, call it something like "command_palette_action" and "Commands..."

**Review by linusg — CHANGES_REQUESTED** [review]
> LibGUI: Don't show the command palette in the command palette
:arrow_down:
> LibGUI: Don't show the command palette **action** in the command palette

File: Userland/Libraries/LibGUI/CommonActions.cpp (hunk 2 above)
Comment (demostanis) [reply Userland/Libraries/LibGUI/CommonActions.cpp]: :+1:

**Pull request merged**